test/render.spec.js

```diff
@@ -0,0 +1,66 @@
+/* eslint-disable no-undef */
+const { generateHtml } = require('@s1seven/schema-tools-generate-html');
+const { generatePdf } = require('@s1seven/schema-tools-generate-pdf');
+const { readFileSync } = require('fs');
+const { resolve } = require('path');
+const { fromBuffer } = require('pdf2pic');
+
+describe('Render', function () {
+  const testSuitesMap = [
+    {
+      certificateName: `valid_certificate_1`,
+    },
```

Comment:
Could you add some new fixtures and declare them here, to make our testing more solid ?

Reply:
Will do.